civictechprojects/static/css/partials/_Global.scss

```diff
@@ -69,3 +69,8 @@ button.clear-button-appearance, a.clear-button-appearance, a[type="button"].clea
     width: 1400px;
   }
 }
+
+.centered {
```

Comment:
We don't need to add this class, Bootstrap has some helper classes available that do the same thing.